### DISCUSSION TITLE
getting this error during installation

### DISCUSSION THREAD

**Discussion by anujverma-prog** [atrocore/atrocore#312] [General]
Hi team,
I am getting below error when i have deployed and trying to open page
 
"GET **/atrocore/api/v1/Installer/getTranslations** HTTP/1.1" 404 239

also in browser

![image](https://user-images.githubusercontent.com/104896491/166654910-cfe4ab7f-f94c-4b8c-afa0-4ec20dcfd774.png)

**Comment by vlad-dc** [comment]
Hello!

I have the same issue, have you been able to solve it??

Regards!

**Comment by rratsun** [comment]
There is no problem here. It looks like your web server configured wrong, that's why you get 404.
Please, read [Installation guide](https://github.com/atrocore/docs/blob/master/atrocore/admin-guide/installation.md) one more time and do the same in your server.

**Comment by anujverma-prog** [comment]
Hi Team,

I am still getting same error even my apache server is working fine. API is not working proper or not able to call , it seems.

![image](https://user-images.githubusercontent.com/104896491/172551820-3d9426fd-8032-4b42-9a91-b2b2e593c7ab.png)

Regards

**Reply by rratsun** [reply]
No. I think your web server configured wrong. Try to set breakpoint into index.php file, then go to ```http://YOUR_HOST/api/v1/Installer/getTranslations```. If you still get 404, then the web server configured wrong. If you get break point, then web server configured correct.

Regards

**Reply by anujverma-prog** [reply]
Thanks for your time, and Yes, you are correct, I think some apache2 server rewrite rule needs to configure.  my project location is /var/www/html/atrocore.   but whenever i access **http://HOST/atrocore**, i am getting this only i.e 404 error.   

**"GET /atrocore/api/v1/Installer/getTranslations HTTP/1.1" 404** 

do you have any configuration of conf/httpd.conf file, so that i can refer

Regards

**Reply by rratsun** [reply]
Please, read [documentation](https://github.com/atrocore/docs/blob/master/atrocore/admin-guide/web-server-preparation-apache.md) again. Here you can find how to enable mod_rewrite and how to configure Virtual Host.

You can also request our paid support or order the [installation service](https://store.atrocore.com/installation-service/037.1).

**Reply by anujverma-prog** [reply]
Thanks a lot for your time, I installed and It's working.
